.github/actions/build/action.yml

```diff
@@ -0,0 +1,20 @@
+name: "Build workspace"
```

Comment:
Please move under `.github/workflows/templates` folder to group them better and rename file to `composite_node_yarn.yml`